app/views/devise/sessions/new.html.erb

```diff
@@ -1,26 +1,41 @@
-<h2>Log in</h2>
+<div class="container-fluid p-0 m-0">
+    <div class="image-container">
+        <div class="wrap-image">
+            <div class="form-container">
+                <%= image_tag 'login-img.png', class: "mx-auto d-block img-fluid" %>
+                <hr>
+                  <% flash.each do |type, msg| %>
+                  <div class="alert alert-info">
+                    <%= msg %>
+                  </div>
+                <% end %>
+                <h4 class="py-4 text-center">Login to continue</h4>
+                <%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
```

Comment:
there's a gem named bootstrap_form_for that wraps all bootstrap forms and make it super easy to create beautiful forms using bootstrap classes

Reply:
`bootstrap_form_for` is a really good gem, but I'd avoid adding more dependencies and would encourage early rails learners to use the helpers that rails already provides.

WDYT? 😄

Reply:
bootstrap_form is listed on README in technology section, we add class: 'form-control' to inputs, it's repetitive, but I think not all team has worked with bootstrap, which option is better in this case?